xml/physical_security.xml

```diff
@@ -426,4 +426,182 @@ polkit.addRule(function(action, subject) {
      </step>
     </procedure>
    </sect1>
+  <sect1 xml:id="sec-USBGuard">
+<title>System protection with enforced USB device authorization via <emphasis>USBGuard</emphasis></title>
+<para>
+  The USBGuard software framework helps to protect your system with enforced USB device authorization.
+  It implements allowlist and blocklist capabilities based on the device attributes.
+</para>
+<para>The USBGuard provides the following features:</para>
+<itemizedlist>
+  <listitem>
+   <para>
+    A command-line interface to interact with a running USBGuard daemon
+    </para>
+  </listitem>
+  <listitem>
+   <para>The daemon component with an inter-process communication (IPC)
+    interface for dynamic interaction and policy enforcement</para>
+  </listitem>
+  <listitem>
+   <para>The rule language for writing USB device authorization policies</para>
+  </listitem>
+  <listitem><para>The C++ API for interacting with the daemon component implemented in a shared library</para></listitem>
+   </itemizedlist>
+   <sect2 xml:id="sec-installing-usbguard">
+    <title>Installing USBGuard</title>
+    <para>You can install and configure USBGuard.</para>
+    <procedure>
+      <step><para>To install USBGuard:</para>
+      <screen>&prompt.sudo; zypper install usbguard</screen>
+    <para>Two packages are installed; <emphasis>usbguard</emphasis> and <emphasis>usbguard-tools</emphasis>.</para>
```

Comment:
This section in now incorrect: usbguard-tools is not a hard dependency, so it will not be installed automatically too.

Maybe rephrase to:
'<emphasis>usbguard</emphasis> and the required dependencies will be installed, if the user needs to interact with the usbguard service then optionally <emphasis>usbguard-tools</emphasis> can be installed too.'

I personally would not make references towards how many packages are installed, because this might change during packaging later on. For example in a later service pack and then might make the documention look outdated to users looking at it. These changes would be probably also very hard to find for the doc team.

Reply:
Removed and makes sense to not mention spec packages

Reply:
Re-phrased.